devel/notcurses/Portfile

```diff
@@ -54,14 +54,13 @@ depends_build-append \
 
 depends_lib-append  path:lib/libavcodec.dylib:ffmpeg \
                     port:libunistring \
+                    port:libdeflate \
                     port:ncurses \
                     port:zlib
```

Comment:
PR description says:

> Drop zlib depend

...but `zlib` is still present in `depends_lib`...

Reply:
good catch, thanks!